Sources/HTMLString/HTMLString.swift

```diff
@@ -140,35 +144,70 @@ extension String {
     /// | `&` | `&` | Not an entity |
     ///
 
-    public mutating func removeHTMLEntities() {
-        var searchSubstring = self[startIndex ..< endIndex]
+    public func removeHTMLEntities() -> String {
+        var result = ""
```

Comment:
Same here with the capacity

```suggestion
        var result = String()
        result.reserveCapacity(count)
```